src/opera/template/operation.py

```diff
@@ -34,6 +34,10 @@ def run(self, host, instance, verbose, workdir):
 
         operation_inputs = {k: v.eval(instance, k) for k, v in self.inputs.items()}
 
+        # TODO(@tadeboro): Currently when primary is None we skip running the operation. Fix this if needed.
```

Comment:
Ha ha, no but since Tadej left maintaining opera to us me and the other contributors kept copying these TODOs. I think that we can get rid of these GitHub user tags in TODO because it's useless.

Reply:
Definitely. Let's at least not add more targeted TODOs in this PR, we'll get around to removing users later :)